component/FloatingAction.js

```diff
@@ -377,7 +377,13 @@ class FloatingAction extends Component {
       <TouchableOpacity
         activeOpacity={1}
         style={[styles.overlay, { backgroundColor: overlayColor }]}
-        onPress={this.reset}
+        onPress={() => {
```

Comment:
Hi @wongjiahau  this will be better to change it for a function and not an inline function... for performance... let's move this into a function called `handleBackdropPress` and then `onPress={this.handleBackdropPress}`

Reply:
Noted.